rhino/src/main/java/org/mozilla/javascript/AbstractEcmaObjectOperations.java

```diff
@@ -232,4 +242,65 @@ static void put(Context cx, Scriptable o, int p, Object v, boolean isThrow) {
             base.put(p, o, v);
         }
     }
+
+    /**
+     * Implement the ECMAScript abstract operation "GroupBy" defined in section 7.3.35 of ECMA262.
```

Comment:
And no need to reference the section number here, as they change from version to version